lib/diagnostic.rb

```diff
@@ -45,7 +56,12 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+class Deverloper < Person
+  def solve_problems
+    puts 'think think think'
```

Comment:
This will return `nil` since `puts` doesn't return a value.  You can use an explicit return with `return 'think think think'` or an implicit return with `'think think think'`.